### ISSUE TITLE
no wiki/docs available

### ISSUE THREAD

**Issue by digitalsignalperson** [threadexio/zeus#3]
The readme has a link to a wiki that just redirects back to the main github page

I came across this from google/reddit but there's not enough info here to know how this tool works.

Qs for some basic info:
- what are the commands to build a package
- what are the outputs
- how does it handle dependencies that are also aur pacakges
- known issues/limitations

**Comment by threadexio** [comment]
Yes, sorry. I am in the middle of rewriting the wiki at this time. And for the rest of your questions you might want to check the CLI help and consult the README here on Github.

> what are the commands to build a package

Simillar to pacman: `zeus -S [package name]`

> what are the outputs

I'm not quite sure what you mean with `outputs`, but the tool builds packages inside a bind-mounted directory at `/var/cache/aur`.

> how does it handle dependencies that are also aur pacakges

It does _not_. Or at least not yet.

> known issues/limitations

The above :point_up: and many other bugs that I have experienced but not documented yet.

I suggest you ignore the version from the `master` branch and instead use the one on the `dev` branch for now. The `master` branch is outdated by a year at this time.
